Misc/NEWS

```diff
@@ -317,6 +317,8 @@ Extension Modules
 Library
 -------
 
+- bpo-30193: Allow to load buffer objects with ``json.loads()``
```

Comment:
You should add a NEWS entry using the blurb tool, and revert this change.

Reply:
Looks like that is being done in @flavianh's follow-up PR ( https://github.com/python/cpython/pull/14977 ).